tests/filecheck/dialects/print/print_to_printf.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: xdsl-opt %s -p print-to-printf | mlir-opt --test-lower-to-llvm | mlir-translate --mlir-to-llvmir | filecheck %s
+// this tests straight to llvmir, because I feel that that's a reasonable thing to do.
```

Comment:
It is in addition to one that doesn't send straight to llvmir :) Also please rephrase the comment to remove the "I", it's a team project

Reply:
Yep! fixed (and added another test)